assets/dash.css

```diff
@@ -0,0 +1,60 @@
+#header {
+  background-color: #000 !important;
+}
+
+  #header a.btn {
+    color: #aaa;
+  }
+
+  #header a.btn:hover {
+    color: #fff;
+  }
+
+
+  #logo {
+    height: 52px;
+  }
+
+
+main .row {
+  margin-bottom: 1.75rem;
+}
+
+
+.tool {
+  display: flex !important;
+  align-items: flex-start !important;
+  border: 1px solid #fff;
+  box-sizing: border-box;
```

Comment:
Nit: Bootstrap puts this on `*` so we can drop it (plus it only has effect if you're setting `width` or `height`)